core/src/main/java/org/apache/iceberg/BaseIncrementalScan.java

```diff
@@ -126,7 +177,6 @@ private Long fromSnapshotIdExclusive(long toSnapshotIdInclusive) {
         // for inclusive behavior fromSnapshotIdExclusive is set to the parent snapshot ID,
         // which can be null
         return table().snapshot(fromSnapshotId).parentId();
-
```

Comment:
nit: unnecessary newline change

Reply:
Done.

Reply:
@hililiwei looks like that newline change is still present